src/extra/bgfx.cpp

```diff
@@ -774,6 +768,7 @@ struct MainWindow : public BaseWindow {
     // set platform data this way.. or we will have issues if we re-init bgfx
     bgfx::PlatformData pdata{};
     pdata.nwh = _sysWnd;
+    // pdata.ndt = SDL_GetNativeDisplayPtr(_window);
```

Comment:
how about this?

Reply:
😅 don't know how I managed to forget the actual fix, thanks